modules/cluster-management/build.gradle

```diff
@@ -63,6 +64,8 @@ dependencies {
     testFixturesImplementation project(':ignite-storage-api')
     testFixturesImplementation project(':ignite-vault')
     testFixturesImplementation project(':ignite-security')
+    testFixturesImplementation project(':ignite-system-disaster-recovery-api')
```

Comment:
It's strange that you need to import both of these modules, why doesn't `ignite-system-disaster-recovery` provide `ignite-system-disaster-recovery-api` transitively?

Reply:
`api project(':ignite-system-disaster-recovery-api')` helped